includes/wp-admin/class-settings.php

```diff
@@ -250,18 +251,9 @@ public static function register_settings() {
 			'activitypub_blog_user_also_known_as',
 			array(
 				'type'              => 'array',
-				'description'       => \__( 'The Account Aliases for the Blog-Actor', 'activitypub' ),
+				'description'       => 'An array of URLs that the blog user is known by.',
 				'default'           => array(),
-				'sanitize_callback' => function ( $value ) {
-					if ( ! is_array( $value ) ) {
-						$value = explode( PHP_EOL, $value );
-					}
-
-					$value = array_map( 'trim', $value );
-					$value = array_map( 'esc_url_raw', $value );
-
-					return array_filter( $value );
-				},
+				'sanitize_callback' => array( Activitypub::class, 'sanitize_textarea_urls' ),
```

Comment:
should we have a global function for that? not sure if this fits best in the ActivityPub class?

Reply:
Yeah, I've been wondering about that, too. I'm not a huge fan of stuffing more things into `functions.php`.  Maybe we could add a class for that? Maybe something that could also cover the register_post_type/meta stuff?